internal/libs/clist/clist.go

```diff
@@ -188,7 +161,7 @@ func (e *CElement) SetPrev(newPrev *CElement) {
 	}
 }
 
-func (e *CElement) SetRemoved() {
+func (e *CElement) setRemoved() {
```

Comment:
This also appears to be used nowhere in the code.

Reply:
Perhaps more broadly, is there any reason not to just delete this methods that appear nowhere again inside of or outside of this package and are not tested?

Reply:
I was hoping that the unused code linter would tell us if they weren't used (though tests can cover things up.)

Reply:
I think this does impact the next waiters, but I pared things down a bunch

Reply:
w00t, looks good.